components/Home/Mech/index.jsx

```diff
@@ -23,7 +20,7 @@ const filterOption = { fromBlock: 28127133, toBlock: 'latest' };
 
 const onNewEvent = (event) => {
   notifySuccess(
-    'New event recevied',
+    'New event received',
```

Comment:
With toasts, we should always use the same pattern.

It should be two words "noun verb" – "Event received" "Message sent" "Profile created" "Message deleted". This pattern works 99% of the time to get the message across in the most efficient way.

Would be good to keep this in mind to save on reviews in the future. Happy to help if you're not sure what it should be.
```suggestion
    'Event received',
```